administration/src/bp-modules/NavigationBar.tsx

```diff
@@ -26,18 +25,28 @@ const NavigationBar = (): ReactElement => {
         <SvgIcon fontSize='large'>
           <EntitlementIcon />
         </SvgIcon>
-        <NavLink to='/' style={{ color: 'inherit', textDecoration: 'none', margin: '0 12px' }}>
+        <Link
+          color='inherit'
+          href='/'
+          underline='none'
+          sx={{ '&:hover': { textDecoration: 'none', color: 'inherit' } }}
+          marginX={1.5}>
           <Stack>
-            <Typography>
+            <Typography variant='body2' component='span'>
               {config.name} {t('administration')}
             </Typography>
-            {!region ? null : (
+            <Stack direction='row' sx={{ gap: 1 }}>
+              {!region ? null : (
+                <Typography variant='body2' component='span'>
+                  {region.prefix} {region.name}
+                </Typography>
+              )}
               <Typography variant='body2' component='span'>
-                {region.prefix} {region.name} {`(${process.env.REACT_APP_VERSION})`}
+                {`(${process.env.REACT_APP_VERSION})`}
```

Comment:
This is an additional change to also show the version for `project-admins` not only if region is available